modules/server/src/network/index.ts

```diff
@@ -35,16 +35,16 @@ const fetchRemoteSchema = async (
 			gqlRequest: { query: getIntrospectionQuery() },
 		});
 
-		if (response.status !== 200) {
-			throw Error('network error');
-		}
-
-		const jsonData = await response.data.json();
-
-		if (jsonData && jsonData.data && isGqlIntrospectionQuery(jsonData.data)) {
-			return { config, introspectionResult: jsonData };
+		if (response.status === 200 && response.statusText === 'OK') {
+			// axios response "data" field, graphql response "data" field
+			const responseData = response.data?.data;
```

Comment:
Glad we're abstracting this within a function, and I wonder if there's a way to remove the second `.data` from the responses (since it's "likely" an Arranger thing anyway)

Reply:
it's a GQL thing

Reply:
GraphQL: here, have some data in your data
![image](https://github.com/overture-stack/arranger/assets/2107110/85762271-dd5d-4787-82b1-5efc79b315c9)